Graph/1466ReorderRoutestoMakeAllPathsLeadtotheCityZero.py

```
@@ -0,0 +1,39 @@
+class Solution: 
+      def __init__(self):
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)